console/program/src/data/record/merkleize.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019-2025 Provable Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use snarkvm_console_algorithms::Poseidon2;
+use snarkvm_console_collections::merkle_tree::PathHash;
+
+use super::*;
+
+impl<N: Network> Record<N, Plaintext<N>> {
+    /// Returns the root of the merkleized record data as defined in `DynamicRecord`
+    pub fn merkleize(
```

Comment:
This can likely be removed since console merkle-ization is implemented [here](https://github.com/ProvableHQ/snarkVM/blob/ce07a2c8edc5f5125b3cc6edac9f920b55b9ca5c/console/collections/src/merkle_tree/mod.rs#L55)